dateutil/rrule.py

```diff
@@ -1697,7 +1697,9 @@ def _parse_rfc(self, s,
                                          value)
                     dtstart = dtvals[0]
                 else:
-                    raise ValueError("unsupported property: "+name)
+                    if re.match("^[A-Za-z0-9-]+$", name):
+                        raise ValueError("unsupported property: " + name)
+                    raise ValueError("RRULE line contains invalid colon or property: " + line)
```

Comment:
This is still pretty cryptic. I haven't read #960 in a while, and I tried running this and got:

```
ValueError: RRULE line contains invalid colon or property: FREQ=YEARLY;WKST=MO;UNTIL=2019-03-29T00:59:59
```

It was very unclear what "invalid colon or property" meant in that situation.

How about instead we just go very wordy instead:

```python
                    if re.match("^[A-Za-z0-9-]+$", name):
                        raise ValueError("unsupported property: " + name)
                    raise ValueError(
                        "RRULE contains invalid property or stray colon:\n"
                        + line + "\n\n"
                        + "This commonly occurs when specifying a datetime "
                        + "using delimiters, e.g. 2025-06-07T12:34:56; RFC "
                        + "5545 requires datetimes specified with no "
                        + "delimiters, e.g. 20250607T123456")
```

Reply:
Yea, this is better; will change accordingly